04-Authorization/.angular-cli.json

```diff
@@ -19,8 +19,8 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "styles": [
-        "styles.css",
-        "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+        "../node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
This sample's showing this error when compiling: 
![image](https://user-images.githubusercontent.com/15782359/38037472-f3e18c74-3276-11e8-966a-12a402a39e6d.png)